wicket-jquery-ui-parent/wicket-jquery-ui-calendar/pom.xml

```diff
@@ -13,8 +13,8 @@
 	<artifactId>wicket-jquery-ui-calendar</artifactId>
 	<packaging>bundle</packaging>
 
-    <properties>
-	    <bundle.namespace>com.googlecode.wicket.jquery.ui.calendar</bundle.namespace>
+	<properties>
+		<bundle.namespace>com.googlecode.wicket.jquery.ui.calendar</bundle.namespace>
```

Comment:
I don't understand why this change shows up like this here. The master branch says `org.wicketstuff.jquery.ui.calendar` here.

Reply:
spaces -> tabs :))) the code mixes these two :(